src/components/filters/filters.js

```diff
@@ -50,22 +51,40 @@ export const Filters = () => {
                                     {eachLabel}
                                 </label> : null
                         ))}
-                    </div>}
+                    </div> : <p>No labels added yet</p>
+}
                     <span>Filter by color</span>
                     <div className="color-filter">
                         {Colors.map((color) => (
                             <label className="note-color" htmlFor={color}
                                 style={{ backgroundColor: color }}
```

Comment:
Could not find the reason of using inline styling. If possible, insert this piece of code into CSS file.

Reply:
Okay